nixos/modules/config/system-path.nix

```diff
@@ -81,6 +81,12 @@ in
         description = "List of additional package outputs to be symlinked into <filename>/run/current-system/sw</filename>.";
       };
 
+      extraSetup = mkOption {
+        type = types.lines;
+        default = [ ];
```

Comment:
Nit: I think using an empty string as the default value is strictly more correct.